packages/mask/background/database/persona/consistency.ts

```diff
@@ -127,6 +127,7 @@ async function* checkProfileLink(
     }
 }
 
+// eslint-disable-next-line @dimensiondev/type/no-const-enum
```

Comment:
I think if the const enum is not exported, it's ok to use. The problem of const enum is that it does not support cross-file transpile, and const enum not exported does not have this problem.